### PR TITLE
feature/cplp 1230/invite company user

### PR DESCRIPTION
**Pull request by oyo**
Note: field validation has been disabled temporarily because of a bug.

### CODE REVIEW

File: cx-portal/src/assets/locales/en/main.json [hunk 1]
```diff
@@ -1046,6 +1046,8 @@
       "INACTIVE": "Inactive",
       "DELETED": "DELETED",
       "name": "Name",
+      "userid": "User ID",
```
Comment (manojava-gk) [inline cx-portal/src/assets/locales/en/main.json:1049]: are they already present in DE?

File: cx-portal/src/components/overlays/AddUser/AddUserDeny.tsx [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+        intro={t(
+          'your action is required before selecting your desired functionality'
+        )}
+        closeWithIcon={true}
+        onCloseWithIcon={() => dispatch(closeOverlay())}
+      />
+      <DialogContent>
+        <div>
+          {t(
+            'Add user not possible because currently you have enabled not exactly one Identity Provider. Before adding users go to the Identity Provider Configuration and choose exactly one IdP where users should log in and disable the others.'
```
Comment (manojava-gk) [inline cx-portal/src/components/overlays/AddUser/AddUserDeny.tsx:52]: better to move to locales

File: cx-portal/src/components/overlays/AddUser/AddUserDeny.tsx [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+          )}
+        </div>
+        <div style={{ marginTop: '30px' }}>
+          {t('The following Identity Providers are enabled')}
```
Comment (manojava-gk) [inline cx-portal/src/components/overlays/AddUser/AddUserDeny.tsx:56]: same here

**Review by manojava-gk — COMMENTED** [review]
Mostly on locales

**Comment by manojava-gk** [comment]
One of the code smells to be fixed

**Review by jjeroch — APPROVED** [review]

File: cx-portal/src/components/overlays/AddUser/AddUserDeny.tsx (hunk 3 above)
Comment (jjeroch) [reply cx-portal/src/components/overlays/AddUser/AddUserDeny.tsx:56]: change will be made after RC3

File: cx-portal/src/components/overlays/AddUser/AddUserDeny.tsx (hunk 2 above)
Comment (jjeroch) [reply cx-portal/src/components/overlays/AddUser/AddUserDeny.tsx:52]: change will be made after RC3

File: cx-portal/src/assets/locales/en/main.json (hunk 1 above)
Comment (jjeroch) [reply cx-portal/src/assets/locales/en/main.json:1065]: change will be made after RC3

**Pull request merged**